General suggestions for nutraceuticals (#815)
greenelab/covid19-review: content/21.nutraceuticals.md

### Abstract

> Coronavirus disease 2019 (COVID-19), caused by the _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2), has caused global disruption and a significant loss of life.

Author: So this is my attempt to adhere to _mSystem_'s formatting guidelines, although the reference list they use doesn't use a lot of the viruses so I'm not totally sure if I got it right!

> Viruses: Names used for viruses should be those approved by  the International  Committee  on  Taxonomy  of  Viruses (ICTV) and reported on the ICTV website. In addition, the recommendations  of  the  ICTV  regarding  the  use  of  species names should generally be followed: when the entire species is discussed as a taxonomic entity, the species name, as with other taxa,  is  italic  and  has  the  first  letter  and  any  proper  nouns capitalized (e.g., _Tobacco mosaic virus_, _Murray Valley encephalitis virus_). When the behavior or manipulation of individual viruses is discussed, the vernacular (e.g., tobacco mosaic virus, Murray Valley encephalitis virus) should be used. If desired, synonyms may be added parenthetically when the name is first mentioned.  Approved  generic  (or  group)  and  family  names may also be used.

From pg 15 on https://msystems.asm.org/sites/default/files/mSys-ITA-12-15-2015.pdf

> Coronavirus disease 2019 (COVID-19), caused by the _Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2), has caused global disruption and a significant loss of life.

Reviewer: Ya maybe go with what they say for now, at the end of the day the reviewers and copy editing can help us there!

Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
A randomized, double-blind, placebo-controlled trial that administered zinc supplementation to elderly subjects over the course of a year found zinc deficiency to be associated with increased susceptibility to infection and that zinc deficiency could be prevented through supplementation [@doi:10.1093/ajcn/85.3.837].
Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
An observational study showed that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls and that zinc-deficient COVID-19 patients (those with levels less than 80 μg/dl) tended to have more complications (70.4% vs 30.0%, _p_ = 0.009) and potentially prolonged hospital stays (7.9 vs 5.7 days, _p_ = 0.048) relative to patients who were not zinc deficient [@doi:10.1016/j.ijid.2020.09.014].

> In coronaviruses specifically, _in vitro_ evidence has demonstrated that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].

Reviewer: Zn2+ should be superscript but I had no idea how to do it

Author: ```suggestion
In coronaviruses specifically, _in vitro_ evidence has demonstrated that the combination of zinc (Zn^2+^) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
```
This will do superscript -- for subscript, it would be `Zn~2+~` (I think! This always throws me!)

These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others.
Indeed, it has been estimated that each degree of latitude north of 28 degrees corresponded to a 4.4% increase of COVID-19 mortality, indirectly linking a persons vitamin D levels via exposure to UVB light to COVID-19 mortality [@doi:10.1111/joim.13149].

As the pandemic has evolved, additional research of varying quality has investigated some of the potential links identified early in the pandemic [@doi:10.3390/nu12040988] between vitamin D and COVID-19. 
Indeed, studies are beginning to investigate whether there is any prophylactic and/or therapeutic relationship between vitamin D and COVID-19.
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; _p_ = 0.004), an association that held when stratifying patients greater than 70 years old [@doi:10.3390/nu12051359].

> These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].

Reviewer: ```suggestion
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].
```

A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.

> <!-- I am a little confused about the next two sentences -- I think they probably did a different comparison but I can't quite work out what it was!-->

Author: Just a note I added for you -- working on the command line is weird!

Reviewer: I think they did, I think looking at the total number of patients there was a significant difference in the 25-hydroxyvitamin D, whereas looking at male and female separately, males were deficient and females were more alike the controls. Does that make sense? :/

Author: Yes! Thank you so much! I might propose a little addition to try to make this a little more obvious!

Reviewer: Perfect :)

> These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].

Author: ```suggestion
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and CT scans of the lungs were obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376].
```

> <!-- I am a little confused about the next two sentences -- I think they probably did a different comparison but I can't quite work out what it was!-->
> Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; _p_ = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
> Evidence of sexual dimorphism was also apparent, as female patients had equivalent levels of 25-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; _p_ = 0.0006).

Author: ```suggestion
Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group as a whole (18.6 ng/ml versus 21.5 ng/ml; _p_ = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%).
However, when comparisons were stratified by sex, evidence of sexual dimorphism became apparent, as female patients had equivalent levels of 25-hydroxyvitamin D to females in the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; _p_ = 0.0006).
```

Reviewer: Yes looks good!

Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.
Despite these concerns, complementary use of probiotics as an adjuvant therapeutic has been proposed by the Chinese National Health Commission and National Administration of Traditional Chinese Medicine [@doi:10.1111/1751-2980.12851]. 
While supply issues prevented the probiotics market from showing the same rapid response to the COVID-19 as some other supplements, many suppliers are reporting growth during the pandemic [@url:https://www.bloomberg.com/press-releases/2020-08-03/probiotics-market-worth-76-7-billion-by-2027-exclusive-report-covering-pre-and-post-covid-19-market-analysis-by-meticulous].
Therefore, the public response once again seems to have adopted supplements promoted as bolstering the immune response despite a lack of evidence suggesting they are beneficial for preventing or mitigating COVID-19.

### Discussion

> In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.

Author: My PhD advisor always liked me to start the discussion with the paragraph that exactly addressed what the paper showed and then move on to look at it in context -- I don't know what you think about moving this paragraph up!

Reviewer: In hindsight I agree with you, a wise supervisor!

As a result, some international authorities have advised the general public, particularly those at high risk of infection, to consider vitamin D supplementation.
However, further well-controlled clinical trials are required to confirm these observations.

Many supplements and nutraceuticals designed for various ailments that are available in the United States and beyond are not strictly regulated [@doi:10.1080/10408398.2019.1592107].
Consequently, there can be safety and efficacy concerns associated with many of these products.
Often, the vulnerable members of society can be exploited in this regard and, unfortunately, the COVID-19 pandemic has proven no different.
As mentioned above, the FDA has issued warnings to several companies for advertising falsified claims in relation to the preventative and therapeutic capabilities of their products against COVID-19 [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-update-fda-and-ftc-warn-seven-companies-selling-fraudulent-products-claim-treat-or].
Further intensive investigation is required to establish the effects of these nutraceuticals, if any, against COVID-19.

> Until more effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html], along with following social distancing measures, "shelter in place" guidelines, expansive testing, and contact tracing [@doi:10.1101/2020.03.27.20045815; @doi:10.3201/eid2608.201093].

Reviewer: ```suggestion
Until more effective therapeutics are established, the most effective mitigation strategies consist of encouraging standard public health practices such as regular hand washing with soap, wearing a face mask, and covering a cough with your elbow [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/prevention.html], along with following social distancing measures, "stay at home" guidelines, expansive testing, and contact tracing [@doi:10.1101/2020.03.27.20045815; @doi:10.3201/eid2608.201093].
```

Reviewer: I think I wrote this and now it sounds like nuclear shelter advice haha. Is stay at home more appropriate?

Author: I think it might be more specific to this situation (I've gotten shelter in place guidelines for other reasons than a pandemic, whereas I hadn't heard of "stay at home" until now!)

Reviewer: sounds good to me :)
